src/multiframe.ts

```diff
@@ -259,17 +259,26 @@ class Multiframe {
             this.totalWeight += sampleWeight;
         }
 
+        // prepare for blit to backbuffer
+        device.setBlending(true);
+        device.setBlendFunctionSeparate(
+            pc.BLENDMODE_SRC_ALPHA,   // SRC_ALPHA,
+            pc.BLENDMODE_ZERO,  // ONE_MINUS_SRC_ALPHA,
```

Comment:
Why are those comments of different blend modes there?

Reply:
this was temporary, been removed.